2_munge/out/agg_nwis.rds.ind

```diff
@@ -1,2 +1,2 @@
-hash: b5908aae7a8167e319f93faa0cd05616
+hash: 6b20d0597dd88345b6d5a266e4c8ef19
 
```

Comment:
This has a conflict with recently merged PR #38 - I think you can just delete this?